### PR TITLE
Added ipfs-config.json

### PR DESCRIPTION
**Pull request by nusserstudios**

### CODE REVIEW

File: ipfs-config.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+				"GET"
+			],
+			"Access-Control-Allow-Origin": [
+				"*"
+			]
+		},
+		"NoDNSLink": false,
+		"NoFetch": false,
+		"PathPrefixes": [],
+		"PublicGateways": null,
```
Comment (itsdevbear) [inline ipfs-config.json:107]: this eventually goes to our domain?

**Comment by itsdevbear** [comment]
@nusserstudios LGTM, I am not familiar with the config though so I trust its right!

**Comment by nusserstudios** [comment]
Yeah, this would be where we configure bongbears.eth, and you can also use the ipfs hash from this as well. I need to figure it out and see how I can grant you rw access so you can either use the cli or web interface to upload stuff. I also figured this would be a really easy copy/paste for the configuration when it gets moved to Dappnode as well.

**Comment by itsdevbear** [comment]
@nusserstudios this fell off my radar, dm me on discord

**Pull request closed without merging**